code/GlobalNavControllerExtension.php

```diff
@@ -6,7 +6,7 @@ public function onBeforeInit() {
 		$host = GlobalNavSiteTreeExtension::get_toolbar_hostname();
 		if (
 				(isset($_REQUEST['regenerate_nav']) || isset($_REQUEST['flush'])) && 
-				$host == Director::absoluteBaseURL() && 
```

Comment:
To further clarify, the line you're linking to was changed to make sure that the other change in get_toolbar_hostname didn't BREAK this code.